tests/h/views/help_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+import mock
+from pyramid import httpexceptions
+
+from h.views import help
+
+
+def test_welcome_page_redirects_to_new_url(pyramid_request, routes):
+    pyramid_request.matched_route.name = "onboarding"
+    result = help.onboarding_page({}, pyramid_request)
+    assert isinstance(result, httpexceptions.HTTPFound)
+
+
+def test_help_page_returns_is_help_true(pyramid_request):
+    pyramid_request.matched_route.name = "help"
+    result = help.help_page({}, pyramid_request)
+    assert result['is_help']
+
+
+def test_custom_welcome_page(pyramid_request, routes):
+    pyramid_request.matched_route.name = "onboarding"
+    result = help.custom_onboarding_page({}, pyramid_request)
+    assert not result['is_help']
+    assert result['is_onboarding']
+
+
+@pytest.fixture
```

Comment:
I don't think any of this is needed any more now that you've split the view functions up nicely.